scripts/mark-definite-framework-results.py

```diff
@@ -32,6 +32,7 @@
 --reassess-failed-draft-services  Don't skip draft_services with "failed" status
 -v, --verbose                     Produce more detailed console output
 --supplier-id-file=<path>         Path to file containing supplier ids to check. One ID per line.
+--excluded-supplier-ids=<esis>    Supplier IDs to be excluded.
```

Comment:
Should probably mention that they're comma separated in case someone thinks there also an "id file" like the option above.

Reply:
Indeed, I will update the README.md with the format required and the _secret_ location of the IDs.